test/built-ins/TypedArray/length-invalid.js

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.2.1.2
+description: TypedArray rejects non-integer arguments
+info: >
+  22.2.1.2 %TypedArray% ( length )
+
+  ...
+  4. Let numberLength be ToNumber(length).
+  5. Let elementLength be ToLength(numberLength).
+  6. ReturnIfAbrupt(elementLength).
+  7. If SameValueZero(numberLength, elementLength) is false, throw a RangeError
+  exception.
+  ...
+features: [TypedArray]
+includes: [testTypedArray.js]
+---*/
+
+assert.throws(RangeError, function() {
```

Comment:
We now have two different versions of the same test, one for ES6 and another for ES7. One of them is going to always fail, but it depends on which version an engine implements.

Given that Test262 is a living test suite (i.e., accepting tests starting from ECMA262 Stage 3), I think we should remove the ES6 version of this test (`test/built-ins/TypedArray/length-invalid.js`). @leobalter, what do you think?